tests/Metadata/Resource/Factory/CachedResourceMetadataFactoryTest.php

```diff
@@ -86,6 +86,6 @@ public function testCreateWithGetCacheItemThrowsCacheException()
 
     private function generateCacheKey(string $resourceClass = Dummy::class)
     {
-        return CachedResourceMetadataFactory::CACHE_KEY_PREFIX.md5(serialize([$resourceClass]));
```

Comment:
This is necessary because the key must not exceed 64 chars (PSR-6). A long class name may exceed this.